### DISCUSSION TITLE
Laravel 9 Support

### DISCUSSION THREAD

**Discussion by disljenko** [livkiss/uniquewith-validator#137] [General]
Dear Maintainers of this library,

is there a timeframe when support with Laravel 9 will implemented. Great library with ton of features and it would be shame not to continue to be supported.

Thanks for all your hard work!

**Comment by Reinbier** [comment]
## Since we've been waiting a long time and no response from the maintainer(s) is received whatsoever:

I, myself, have used this package for a long time as well and migrated my projects over to Laravel 9 and now Laravel 10 is knocking on my door as well. I've copied the source and continued this package by myself the past year. 

Since I sense there is enough support for this project to keep it alive, I will gladly make my code available to the public. 
It works the same, only L9+L10 are now supported. (Not sure about backwards compatibility since I did not test beyond L9)

### You can find it here: https://github.com/Reinbier/laravel-unique-with

I'd sure appreciate anyone who can help me keep this code alive and available to everyone who uses the Laravel Framework.
